htdocs/accountancy/class/accountancyexport.class.php

```diff
@@ -718,7 +718,7 @@ public function exportFEC($objectLines)
 			print $line->piece_num.$separator;
 
 			// FEC:EcritureDate
-			print $date_creation.$separator;
+			print $date_doc.$separator;
```

Comment:
@aspangaro Do you agree with this change ?

Reply:
I don't think it should be included. My accountant complained that the dates were wrong, which is why I initially pushed this change. However, it seems the problem was in the date format (see 72b7e72) and not the order of the fields.

I reverted this change in b7e4d49.

Sorry for pushing several unrelated changes to the same PR, I'll make sure to separate them better in the future.